### DISCUSSION TITLE
Full live reload (C# and UI)

### DISCUSSION THREAD

**Discussion by mrNo0b** [dotnet/maui#291] [Ideas]
Would be nice if it will be possible to have full live reload for both C# code and UI, 
I use WPF and when I change C# code I have to recompile the app and restart. it takes time,
Even if it will mean slower debug mode app performance I guess...

**Comment by realtebo** [comment]
Microsoft is aiming to remove Jw from web development. I hope it will happen. And to runaway from actual developers heaven like quasar we developers need the hot reload and possibly no compile but runtime only in development .

**Comment by edsulaiman** [comment]
I have been using xamarin forms for 6 months, i always recompile it every time i make a change to logical code. I hope hot reloading on xamarin forms will apply to all changes made to files in a project
